examples/pytorch/question-answering/utils_qa.py

```diff
@@ -70,10 +70,10 @@ def postprocess_qa_predictions(
             answers, are saved in `output_dir`.
         prefix (:obj:`str`, `optional`):
             If provided, the dictionaries mentioned above are saved with `prefix` added to their names.
-        log_level (:obj:`str`, `optional`, defaults to ``passive``):
-            Logger log level to use on the main process. Possible choices are the log levels as strings: 'debug',
-            'info', 'warning', 'error' and 'critical', plus a 'passive' level which doesn't set anything and lets the
-            application set the level.
+        log_level (:obj:`int`, `optional`, defaults to ``logging.WARNING``):
+            log level to be used depending on whether this process is the main process of node 0, main process
+            of node non-0, or a non-main process. Possible choices are the log levels as int: 'logging.WARNING',
+            'logging.INFO' etc depending on `TrainingArguments`
```

Comment:
since we removed the logic of figuring out which log level to use when, and having the caller figure it out instead, I'd say all we need here is just:

```suggestion
        log_level (:obj:`int`, `optional`, defaults to ``logging.WARNING``):
            ``logging`` log level (e.g. ``logging.WARNING``)
```

Reply:
same in the other place below.